index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html lang="en-US">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <title>Span Registration Page</title>
+    <link rel="stylesheet" type="text/css" media="screen" href="style.css" />
+  </head>
+  <body>
+    <div class="form-style-10">
+      <h1>
+        Microverse Entry Trial Survey!<span
+          >Sign up for the 6 - 8 months software training program</span
+        >
+      </h1>
+
+      <form>
+        <div class="section"><span>1</span>Name, Email &amp; Age</div>
+        <div class="inner-wrap">
+          <label>Name<sup>**</sup> <input type="text" /> </label>
+          <label>Email<sup>**</sup> <input type="email" /></label>
+          <label>Age <input type="number" /></label>
+        </div>
+
+        <div class="section"><span>2</span>Other Details</div>
+        <div class="inner-wrap">
+          <label>
+            Which option best describes your current role?
+            <select>
+              <option>Select current role</option>
+              <option>Student</option>
+              <option>Full Time Job</option>
+            </select>
+          </label>
+
+          <label>Would you recommend freeCodeCamp to a friend? </label>
+          <p>
+            <label><input type="radio" name="option" /> Definitely</label>
+          </p>
+          <p>
+            <label><input type="radio" name="option" /> Maybe</label>
+          </p>
+          <p>
+            <label><input type="radio" name="option" /> Not Sure</label>
+          </p>
+
+          <label> What is your favorite feature of freeCodeCamp? </label>
+          <select>
+            <option>Select an option</option>
+            <option>Challenges</option>
+            <option>Project</option>
+            <option>Community</option>
+          </select>
+
+          <label>
+            What would you like to see improved? (Check all that apply)
+          </label>
+          <p>
+            <label><input type="checkbox" /> Front-end Projects</label>
+          </p>
+          <p>
+            <label><input type="checkbox" /> Back-end Projects</label>
+          </p>
+          <p>
+            <label><input type="checkbox" /> Machine Learning</label>
+          </p>
+
+          <label>
+            Any comments or suggestions?
+            <textarea></textarea>
+          </label>
+        </div>
+
+        <div class="button-section">
+          <input type="submit" value="Submit" />
+        </div>
+      </form>
+    </div>

```

Comment:
- [ ] I think it will be better if you can use [`HTML Semantic Tags`](https://www.w3schools.com/html/html5_semantic_elements.asp). This will ensure that your webpage content is easily accessible by technologies like screen readers. Also, semantic elements clearly describe its meaning to both the browser and the developer.

Reply:
It will be very nice if you add some emojis for your inline comments and reviews, emojis always make your review more human-friendly and make you look like a kind person. :smile:  :hugs: .,

And also it is always a good practice to look for `Bugs` in the first build of your project, this will ensure that the project set up correctly :rocket: :100:
 also, it will be very cool if you add some suggestions for improvement and new features. :smile: :cat: